actions/docs-verifier/src/RedirectionVerifier/DocfxConfigurationReader.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Text.Json;
+using Microsoft.Extensions.FileSystemGlobbing;
+
+namespace RedirectionVerifier
+{
+    public static class DocfxConfigurationReader
```

Comment:
I'll look into having a base abstract "ConfigurationReader" class that simplifies the multiple readers we currently have and extract common logic into it.

Reply:
I'd be happy to write that part, once these all get in. I created an issue to track it here #23